deps/db/src/logseq/db/frontend/property.cljs

```diff
@@ -500,6 +500,11 @@
                                                       :hide? true
                                                       :public? false}
                                              :rtc property-ignore-rtc}
+     :logseq.property.asset/last-visit-scale {:title "Last visit scale"
```

Comment:
need to add a new migration in `frontend.worker.db.migrate/schema-version->updates`

see also prompts/review.md:
> - After adding a new property in `logseq.db.frontend.property/built-in-properties`, you need to add a corresponding migration in `frontend.worker.db.migrate/schema-version->updates`.
>  - e.g. `["65.9" {:properties [:logseq.property.embedding/hnsw-label-updated-at]}]`